README.md

```diff
@@ -11,4 +11,58 @@ read the docs badge - [![Documentation Status](https://readthedocs.org/projects/
      - For more information on how this works, checkout the mkdocs website - https://mkdocs.org
      - Specifically, we are using the 'material mkdocs' theme, described here - https://squidfunk.github.io/mkdocs-material/
 
-test
+___
+# Style Guide
+
+This is a work in progress :)
+
+The goal is to create a consistent style across all parts of the docs to create what I, Trent, am calling "semantic continuity" in how we communicate to our users. Semantic continuity means that the meaning of various symbols and text formatting is consistent across our documentation.
+- This could be as small as making sure that there are periods at the end of every sentence, even in bullet points.
+- Or it could be as big as making sure that [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/), or "call-outs", are used in the same way across the docs.
+
+## [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) - AKA Call-Outs
+Our docs currently employ `6` types of Admonitions. The four which we have customized which can be found in `/stylesheets/extra.css`, and the other standard styles of admonitions can be found in [here](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types).
+
+Each of the Admonitions have a unique aesthetic and serve a specific purpose:
+
+`tip-full-width`
```

Comment:
I think it scans better when the 'admonition' is closer to the label. 

You could also try using a 3rd level heading (`###`) for each one to see how that looks. 

You could also potentially add info in th e text the admonition image itself, but then it would be hard to alter/edit in the future